packages/admin-panel/src/pages/resources/SocialFeedPage.js

```diff
@@ -28,10 +28,7 @@ const FIELDS = [
     Header: 'Creation date',
     source: 'creation_date',
     type: 'tooltip',
-    accessor: row =>
-      moment(row.creation_date)
-        .local()
-        .toString(),
+    accessor: row => moment(row.creation_date).local().toString(),
```

Comment:
Prettier 2 (which we now use) uses an, according to them, [Improved method chain breaking heuristic](https://prettier.io/blog/2020/03/21/2.0.0.html#improved-method-chain-breaking-heuristic-6685httpsgithubcomprettierprettierpull6685-by-mmkalhttpsgithubcommmkal). They only break chained methods in new lines if they are deemed to be complicated.

Is this new style better than the previous one? It's a matter of taste and I'm not 100% sure, but I think I like it

Reply:
yeah I personally prefer this new style as it reads more like a sentence